website/docs/reference/resource-configs/fabric-configs.md

```diff
@@ -85,6 +85,10 @@ Indexes are not supported by Microsoft Fabric Synapse Data Warehouse. Any Indexe
 
 Grants with auto provisioning is not supported by Microsoft Fabric Synapse Data Warehouse at this time.
 
+## Incremental
+
+Fabric supports the `delete+insert` strategy. If a unique key is not provided, it will default to the `append` strategy
```

Comment:
I'm confused -- is this what you mean? Or do you mean to say: "if you specify `strategy="delete+insert"` without a primary key, it will actually just do `insert` which amounts to an `append` strategy.

```suggestion
Fabric supports both `delete+insert` and `append` strategy.

If a unique key is not provided, it will default to the `append` strategy
```